components/resource_control/src/metrics.rs

```diff
@@ -4,22 +4,25 @@ use lazy_static::*;
 use prometheus::*;
 
 lazy_static! {
+    // TODO: deprecate the "name" label in v8.0.
```

Comment:
I see some metrics not introduced in the Grafana template. maybe we can directly rename some metrics in `TiKV`

Reply:
All the metrics changed in this PR are used in grafana. The background related metrics panels are defined in the Resource-Control page in tidb repo.